# Remove hard coeded xHost flag, slows things down on AMD CPUs, and arch flag is anyway set by EB
# March 2nd 2021 By B. Hajgato (UGent)
--- MMseqs2-11-e1a1c/src/CMakeLists.txt.orig	2020-02-11 17:17:38.000000000 +0100
+++ MMseqs2-11-e1a1c/src/CMakeLists.txt	2021-03-02 11:29:22.146842737 +0100
@@ -60,8 +60,8 @@
 
 if (CMAKE_BUILD_TYPE MATCHES RELEASE OR CMAKE_BUILD_TYPE MATCHES RELWITHDEBINFO)
     if (CMAKE_COMPILER_IS_ICC)
-        append_target_property(mmseqs-framework COMPILE_FLAGS -ipo -no-prec-div -xHost)
-        append_target_property(mmseqs-framework LINK_FLAGS -ipo -no-prec-div -xHost)
+        append_target_property(mmseqs-framework COMPILE_FLAGS -ipo -no-prec-div )
+        append_target_property(mmseqs-framework LINK_FLAGS -ipo -no-prec-div )
     else ()
         append_target_property(mmseqs-framework COMPILE_FLAGS -ffast-math -ftree-vectorize -fno-strict-aliasing)
         append_target_property(mmseqs-framework LINK_FLAGS -ffast-math -ftree-vectorize -fno-strict-aliasing)
